### ISSUE TITLE
[dependencies] Detailed dependency list

### ISSUE THREAD

**Issue by kunaltyagi** [crosstool-ng/crosstool-ng#1344]
[Webpage](https://crosstool-ng.github.io/docs/install/#install-method) doesn't list the dependencies. I've compiled my own list of dependencies, however, I'm not sure where to add them.

Moreover, I'm not sure which of them are solely build-dependencies, and which are run-time dependencies also.

Dependency list:
```
autoconf \
        bison \
        build-essential \
        flex \
        gawk \
        gettext \
        g++ \
        help2man \
        libncurses-dev \
        libtool-bin \
        texinfo \
        unzip
```
I installed git and cmake for my reasons and those might be masking other dependencies.

**Comment by cpackham** [comment]
It is a little hard to maintain a complete list for every linux distro (although generally all these packages have the same name except perhaps ncurses). Another source for this list is the `Dockerfile`s under https://github.com/crosstool-ng/crosstool-ng/tree/master/testing/docker

**Comment by kunaltyagi** [comment]
> It is a little hard to maintain a complete list for every linux distro

I agree. However, would it be possible to have a separate runtime dependency for any distro? The ones on the Dockerfiles are all compile time dependencies.
